.travis.yml

```diff
@@ -1,4 +1,6 @@
-language: generic # don't install any environment
+language: python # don't install any environment
```

Comment:
The comment becomes outdated.

Reply:
@Totktonada yes, but Travis doesn't run the builds anyway, so I'm going to close this PR. We'll probably have to rebuild CI on GH Actions.